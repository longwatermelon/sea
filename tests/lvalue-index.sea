fn main() -> int {
    let arr:int* = stalloc(3, int);
    arr[0] = 2;
    arr[1] = 3;
    arr[2] = 5;

    let ptr:int* = &arr[1];
    *ptr = *ptr*2;

    return arr[1];
}
