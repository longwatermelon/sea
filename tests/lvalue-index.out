RETURN_CODE:6
STDOUT:
